packages/um/package.py

```diff
@@ -329,26 +395,30 @@ def _build_dir(self):
 
 
     def patch(self):
-        """Patch the staging directory just before building."""
-        
-        # Get the root to the resources
-        resources_root = join_path(self.stage.source_path, "resources")
-
-        # Optional UM sources (i.e. AM3)
-        if self.spec.variants["um_sources"].value != "NA":
-            self._dynamic_resource(
-                url=self.um_git_url,
-                ref=self.spec.variants["um_sources"].value,
-                dst_dir=join_path(resources_root, "um")
-            )
-
-        # Optional JULES sources (i.e. AM3)
-        if self.spec.variants["jules_sources"].value != "NA":
-            self._dynamic_resource(
-                url=self.jules_git_url,
-                ref=self.spec.variants["jules_sources"].value,
-                dst_dir=join_path(resources_root, "jules")
-            )
+        """
+        Patch the staging directory just before building.
+        """
+
+        # This patch is relevant only for models that use Github URLs.
+        # Only one model so far, but this may change in future.
+        spec = self.spec
+        model = spec.variants["model"].value
+        if model == "vn13p1-am":
+            # Get the root to the resources
+            resources_root = join_path(self.stage.source_path, "resources")
+
+            # Optional sources (i.e. AM3)
+            for var in self._resource_cfg:
```

Comment:
Perhaps use the same variable name for iterating over `self._resource_cfg` as you have done elsewhere.